imjoy/imjoyPluginEngine.py

```diff
@@ -793,7 +804,7 @@ def connect(sid, environ):
                 f"starting background task with command `{cmd}` to continously read "
                 "and forward pty output to client"
             )
-            logger.debug("xterm task started", terminal_session)
+            logger.debug("xterm task %s started", terminal_session)
```

Comment:
Thanks for fixing.

`terminal_session` is actually a dictionary, would it be converted to string?

Reply:
Yes, a dict has a magic method `__str__`. It's the same as if you would print a dict.

Reply:
Sounds good. Please merge when you are ready.

Reply:
```
In [1]: test = {'one': 1}                                                                                                                                                                                     

In [2]: import logging                                                                                                                                                                                        

In [3]: logging.basicConfig(level=logging.DEBUG)                                                                                                                                                              

In [4]: logging.debug("test %s", test)                                                                                                                                                                        
DEBUG:root:test {'one': 1}
```

Reply:
We can remove the argument, if it wasn't intended to be logged.

Reply:
Let's keep it, it would be helpful to log.

And there are a few other places which I didn't add log. Let's fix them in future PRs.